cryptor/cryptor.go

```diff
@@ -742,6 +742,11 @@ var ErrRestoreDelegations = errors.New("cryptor: need more delegations")
 // enough delegations are present to restore the cache, the current
 // Red October key cache is replaced with the persisted one.
 func (c *Cryptor) Restore(name, password string, uses int, slot, durationString string) error {
+	// If the persistence store is already active, don't proceed.
+	if st := c.persist.Status(); st != nil && st.State == persist.Active {
+		return nil
```

Comment:
My main reason for not making it an error is that errors trigger a red
warning banner, and this doesn't seem like a case where that kind of user
feedback is necessary. Instead, the UI will show a green banner with the
current persistence status.